pkg/profile/v1-profile.json

```diff
@@ -184,45 +79,27 @@
                 }
               },
               "additionalProperties": false,
-              "required": [
-                "r",
-                "g",
-                "b"
-              ]
+              "required": ["r", "g", "b"]
             }
           },
           "additionalProperties": false,
-          "required": [
-            "color"
-          ]
+          "required": []
         },
-        "eyebrows": {
-          "type": "object",
-          "properties": {
-            "texture": {
-              "type": "string"
-            }
+        "bodyShape": {
+          "type": "string"
+        },
+        "wearables": {
+          "type": "array",
+          "items": {
+            "type": "string"
           },
-          "additionalProperties": false,
-          "required": [
-            "texture"
-          ]
+          "additionalItems": false
         }
       },
       "additionalProperties": false,
-      "required": [
-        "version",
-        "id",
-        "bodyShape",
-        "eyes",
-        "mouth",
-        "wearables",
-        "skin",
-        "hair",
-        "eyebrows"
-      ]
+      "required": ["version", "skin", "hair", "bodyShape", "wearables"]
     }
   },
   "additionalProperties": false,
-  "required": []
-}
\ No newline at end of file
+  "required": ["name", "description", "created_at", "avatar"]
```

Comment:
created_at? why?

Reply:
It's part of the passport, It represents since when the user is part of DCL